one_fm/public/js/doctype_js/job_applicant.js

```diff
@@ -1138,6 +1143,10 @@ var set_work_details_section = function(frm, erf) {
 
 var set_erf_basic_details = function(frm, erf) {
 	frm.set_value('one_fm_hiring_method', erf.hiring_method);
+	if(erf.working_days && erf.off_days && erf.off_days > 0){
+		frm.set_value('day_off_category', 'Monthly');
```

Comment:
@pjamsheer Just Curious! Why are you setting "day_off_category" & "number_of_days_off" twice?

Reply:
@SalShai Setting Day Off Category from client side for who creates Job Applicant from Desk and Setting Day Off Category from validate of Job Applicant for who creates Job Applicant from the portal or through API access.